facebook/__init__.py

```diff
@@ -31,6 +31,7 @@
 import requests
 import json
 import re
+from enum import Enum
```

Comment:
This import isn't Python 2.7 compatible.

Reply:
Fixed in https://github.com/mobolic/facebook-sdk/pull/463/commits/b11fe55561d078fd3e3f91b9b09c92fc4f99e611